app/models/ems_refresh.rb

```diff
@@ -17,7 +17,7 @@ def self.debug_trace
   cache_with_timeout(:queue_timeout) { MiqEmsRefreshWorker.worker_settings[:queue_timeout] || 60.minutes }
 
   def self.queue_refresh_task(target, id = nil)
-    queue_refresh(target, id, :create_task => true)
+    queue_refresh(target, id, :create_task => true, :action => "EmsRefresh completed successfully")
```

Comment:
@kbrock will this say `"EmsRefresh completed successfully"` while it is still running?  If it failed?
Would `:action => "Refreshing relationships and power states"` or something like that make more sense?  Maybe I'm missing when/where the `:action` attribute is displayed

Reply:
Yea, don't know when task action is displayed, @Fryguy ?